phpstan.neon.dist

```diff
@@ -0,0 +1,45 @@
+parameters:
+  parallel:
+    maximumNumberOfProcesses: 1
+  level: 9
+  paths:
+    - web/modules/custom
+  customRulesetUsed: true
+  ignoreErrors:
+    - '#Function template_[a-zA-Z\\_\\\\(\\)]+ has parameter [a-zA-Z\\$\\_]+ with no value type specified in iterable type array#'
```

Comment:
That's not needed

Reply:
+

Reply:
Did you remove just one line here? @OleksandrRiumshyn

Reply:
@AndreyMaximov I've removed 2 lines and leave in place these:
   ` #Function [a-zA-Z0-9\\_\\\\:\\(\\)]+ return type has no value type specified in iterable type array#`
This one needs to avoid problems with Drupal alters and hooks (Implements comment)
   ` #Method Drupal[a-zA-Z0-9\\_\\\\:\\(\\)]+ has parameter \$[a-zA-Z0-9\\_]+ with no value type specified in iterable type array#`
    `#Method Drupal[a-zA-Z0-9\\_\\\\:\\(\\)]+ return type has no value type specified in iterable type array#`
These two needs to avoid problems with calling of Drupal services then impossible to make dependency injection